pages/common/git-annex.md

```diff
@@ -0,0 +1,28 @@
+# git annex
+
+> Manage files with git, without checking their contents in.
+> When  a  file  is annexed, its content is moved into a key-value store, and a symlink is made that points to the content. More information: <https://git-annex.branchable.com/>.
+
+- Help:
+
+`git annex help`
```

Comment:
I think this command is handled by the `git help {{command}}` on the page for `git` itself. What do you think?

Reply:
git help annex will give you the manpage, while the git annex help will give a list of 9 most frequently used commands ... while manpage is probably more useful git annex help is more specific and may be better to learn from a tldr

Reply:
@sbrl This is because `annex` is like a plugin for git.

Reply:
Ah, I see. Nevermind then :-)